tests/installation/addon_products_sle.pm

```diff
@@ -36,7 +36,6 @@ sub run() {
             wait_still_screen 3;
             send_key $cmd{next};
             assert_screen 'dvd-selector';
-            send_key_until_needlematch 'addon-dvd-list',         'tab',  10;    # jump into addon list
```

Comment:
Does it really harm to ensure that one is the right window?
Does it still work with SP1?